libraries/AP_SerialManager/AP_SerialManager.h

```diff
@@ -131,7 +131,7 @@ class AP_SerialManager {
         SerialProtocol_Sbus1 = 15,
         SerialProtocol_ESCTelemetry = 16,
         SerialProtocol_Devo_Telem = 17,
-        SerialProtocol_OpticalFlow = 18,
```

Comment:
this change to AP_SerialManager shouldn't be necessary.  the FLOW_TYPE allows the user to specify the type, that doesn't need to be done again using the SERIALx_PROTOCOL parameter.

Reply:
Deleted the line.